src/coreclr/src/inc/utilcode.h

```diff
@@ -1280,16 +1280,16 @@ class NumaNodeInfo
 
     static LPVOID VirtualAllocExNuma(HANDLE hProc, LPVOID lpAddr, SIZE_T size,
                                      DWORD allocType, DWORD prot, DWORD node);
-#ifndef TARGET_UNIX
+#ifndef HOST_UNIX
```

Comment:
Nit: I would be nice to change all `#ifndef HOST_UNIX` and `#if !defined(HOST_UNIX) ....` in this delta to `#ifdef HOST_WINDOWS`.

Reply:
Ok, I see that I have commented on similar one already. The following pattern catches my eye:

```
#if !UNIX
.... calls to very Windows specific APIs that won't ever exist anywhere else but on Windows ...
#endif
```